observability/otel/config.ts

```diff
@@ -29,6 +29,16 @@ const tryGetVersionOf = (pkg: string) => {
 const apps_ver = tryGetVersionOf("apps/") ??
   tryGetVersionOf("deco-sites/std/") ?? "_";
 
+const getCloudProvider = () => {
+  const kService = Deno.env.get("K_SERVICE") !== undefined 
+
+  if (kService) {
+    return "AWS";
+  } else{
```

Comment:
fix formatting

Reply:
import { context } from "../../live.ts";
```suggestion
  } else if (context.isDeploy) {
```